packages/cfpb-expandables/src/summary.less

```diff
@@ -80,8 +80,8 @@
       top: calc(-100% + 4px);
       background: linear-gradient(
         to bottom,
-        rgba(255, 255, 255, 0) 0%,
-        rgba(255, 255, 255, 1) 100%
+        rgb(255 255 255 / 0%) 0%,
+        rgb(255 255 255 / 100%) 100%
```

Comment:
This fix is from the `modern` config in https://stylelint.io/user-guide/rules/color-function-notation/